Core/TemplateLoader.py

```diff
@@ -10,10 +11,9 @@ def __init__(self, pathList, **kwargs):
     self.pathList = pathList
 
   def resolve_path(self, name, parent_path=None):
-    if parent_path and not parent_path.startswith("<") and \
-            not parent_path.startswith("/") and \
-            not name.startswith("/"):
-      name = os.path.join(os.path.dirname(parent_path), name)
+    if parent_path and not parent_path.startswith("<"):
+      if not parent_path.startswith("/") and not name.startswith("/"):
+        name = os.path.join(os.path.dirname(parent_path), name)
```

Comment:
Why this change is needed?  The logic has not changed. No?

Reply:
This change resolve format check error

Reply:
ok.